MANIFEST.in

```diff
@@ -1,6 +1,4 @@
 prune common
 include LICENSE
-include sphinx_search/_static/js/rtd_sphinx_search.js
 include sphinx_search/_static/js/rtd_sphinx_search.min.js
-include sphinx_search/_static/css/rtd_sphinx_search.css
```

Comment:
I do think we should include these as well as the minified ones. People generally like to be able to debug against non-minified source.